don't install these symlinks, it's not pyside2-tools business:
* /usr/bin/designer -> /usr/bin/qtchooser
* /usr/bin/rcc -> /usr/bin/qtchooser
* /usr/bin/uic -> /usr/bin/qtchooser

--- a/sources/pyside2-tools/CMakeLists.txt
+++ b/sources/pyside2-tools/CMakeLists.txt
@@ -55,38 +55,6 @@
     set(DESIGNER_PATH "${TOOLS_PATH}/designer${EXE_EXT}")
 endif()
 
-install(FILES "${UIC_PATH}"
-        DESTINATION bin
-        PERMISSIONS
-        OWNER_EXECUTE OWNER_WRITE OWNER_READ
-        GROUP_EXECUTE GROUP_READ
-        WORLD_EXECUTE WORLD_READ)
-
-install(FILES "${RCC_PATH}"
-        DESTINATION bin
-        PERMISSIONS
-        OWNER_EXECUTE OWNER_WRITE OWNER_READ
-        GROUP_EXECUTE GROUP_READ
-        WORLD_EXECUTE WORLD_READ)
-
-if (EXISTS ${DESIGNER_PATH})
-    if (APPLE)
-        install(DIRECTORY "${DESIGNER_PATH}"
-                DESTINATION bin
-                FILE_PERMISSIONS
-                OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                GROUP_EXECUTE GROUP_READ
-                WORLD_EXECUTE WORLD_READ)
-    else()
-        install(FILES "${DESIGNER_PATH}"
-                DESTINATION bin
-                PERMISSIONS
-                OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                GROUP_EXECUTE GROUP_READ
-                WORLD_EXECUTE WORLD_READ)
-    endif()
-endif()
-
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake_uninstall.cmake"
                "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
                IMMEDIATE @ONLY)
